andromeda/run.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+gunicorn -w $BACKEND_WORKERS -b :5000 -t 360 wsgi:app
```

Comment:
@thompsonmj So you are suggesting this:
```suggestion
gunicorn -w ${BACKEND_WORKERS:=4} -b :5000 -t 360 wsgi:app
```